src/SourceLocator/Type/EvaledCodeSourceLocator.php

```diff
@@ -22,11 +22,11 @@ final class EvaledCodeSourceLocator extends AbstractSourceLocator
     /** @var SourceStubber */
     private $stubber;
 
-    public function __construct(Locator $astLocator)
+    public function __construct(Locator $astLocator, SourceStubber $stubber)
```

Comment:
This change is a major BC break - should we mitigate or target the next major release?

Reply:
BC solution is to make it optional (potentially deprecating omitting it) and instantiate a ReflectionSourceStubber when omitted (which will correspond to the old behavior)

Reply:
I've made it optional currently - it looks like good solution for me.